tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Constraints.cpp

```diff
@@ -372,6 +372,31 @@ void test_four_index_constraint_analytic(
       make_with_value<decltype(four_index_constraint)>(x, 0.0),
       numerical_approx);
 }
+
+// Test the return-by-value constraint energy function using random values
+// Note: this function calls the return-by-reference constraint energy
+// function, so both return-by-value and return-by-reference functions
+// are tested here. Note that testing also against Kerr-Schild would be
+// redundant here, since each constraint is already tested against
```

Comment:
I disagree. Please add a Kerr-Schild test.

Reply:
You've added the Kerr-Schild test. When you rebase after the F constraint is merged could you remove that part of this comment?